src/main/kotlin/com/github/biomejs/intellijbiome/services/BiomeServerService.kt

```diff
@@ -68,49 +68,32 @@ class BiomeServerService(private val project: Project) {
         val commandName = BiomeBundle.message("biome.run.biome.check.with.features",
             features.joinToString(prefix = "(", postfix = ")") { it -> it.toString().lowercase() })
 
-        val formattingActions = mutableListOf<Runnable>()
-
         if (features.contains(Feature.ApplySafeFixes) || features.contains(Feature.SortImports)) {
-            val result = mutableListOf<LspIntentionAction>()
-
+            val actions = mutableListOf<String>()
             if (features.contains(Feature.ApplySafeFixes)) {
-                val codeActionParams = CodeActionParams(server.getDocumentIdentifier(file),
-                    getLsp4jRange(document, 0, document.textLength),
-                    CodeActionContext().apply {
-                        diagnostics = emptyList()
-                        only = listOf<String>("quickfix.biome")
-                        triggerKind = CodeActionTriggerKind.Automatic
-                    })
-
-                val codeActionResults = server.sendRequest { it.textDocumentService.codeAction(codeActionParams) }
-                codeActionResults?.forEach {
-                    if (it.isRight) {
-                        result.add(LspIntentionAction(server, it.right))
-                    }
-                }
+                actions.add("quickfix.biome")
             }
-
             if (features.contains(Feature.SortImports)) {
-                val codeActionParams = CodeActionParams(server.getDocumentIdentifier(file),
-                    getLsp4jRange(document, 0, document.textLength),
-                    CodeActionContext().apply {
-                        diagnostics = emptyList()
-                        only = listOf<String>("source.organizeImports.biome")
-                        triggerKind = CodeActionTriggerKind.Automatic
-                    })
-
-                val codeActionResults = server.sendRequest { it.textDocumentService.codeAction(codeActionParams) }
-                codeActionResults?.forEach {
-                    if (it.isRight) {
-                        result.add(LspIntentionAction(server, it.right))
-                    }
-                }
+                actions.add("source.organizeImports.biome")
```

Comment:
Do you know if there could be a potential problem with the following case:
	1.	A quick fix returns text edits for imports.
	2.	Sorting imports changes the order.
	3.	The quick fix produces incorrect text edits (or sorting imports without the quick fix).

Reply:
Hm,
I recall that the some versions of LSP server do not return edits for sorting and quick fixes simultaneously. 
Probably we need to apply sorting and after that request quickfixes (or viceversa)?

Reply:
Even if the LSP server returns import sorting and quick fixes simultaneously, it's okay if the text edits are in a correct order. I tested some cases with quick fixes + sort imports in this branch, it's working perfectly for me. If you have a case that breaks a quick fix or import sorting, I'll revert that part to run the two actions separately.

Reply:
I reverted a part of the changes to just run the two actions separately.